### PR TITLE
add g-release to quick start guide

### LINKED ISSUE #106: Add Greenland release to Quick Start Guide
**Issue/Feature Description:**
Add Greenland release to quick start guide. 

**Why this issue to fixed / feature is needed(give scenarios or use cases):**
To Add latest release to the install guide. 

**How to reproduce, in case of a bug:**
NA
**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**
NA

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
 /kind documentation

**What this PR does / why we need it**:
Adds G-release version to the quick start guide. 

**Which issue(s) this PR fixes**:

Fixes #106 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED

**Test Report**:

image
image

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
@anvithks @kumarashit  Please merge

**Comment by kumarashit** [comment]
@rajat-soda This will remove the link for Faroe release. That's not correct as Faroe release notes still will refer to old link and will break. Can we add new section? 
Or a better solution is to put the version number as VARIABLE and keep just one text. And just update the release tag mapping.
Ex:
**********************
|RELEASE NAME | RELEASE TAG|
|Greenland | v1.1.0|
|Faroe | v1.0.0|
|Elba | v0.20.0|

Install SODA Release
Quick installation from SODA Release:

Prerequisite

OS: Ubuntu 16.04 or Ubuntu 18.04

root user rights is REQUIRED
.......
......
.......

Get and Install Release Binaries

wget https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-<RELEASE_TAG>.tar.gz OR Download the installer binaries from
https://github.com/sodafoundation/installer/releases/tag/<RELEASE_TAG>

**Comment by rajat-soda** [comment]
> @rajat-soda This will remove the link for Faroe release. That's not correct as Faroe release notes still will refer to old link and will break. Can we add new section?
> Or a better solution is to put the version number as VARIABLE and keep just one text. And just update the release tag mapping.
> Ex:
> 
> |RELEASE NAME | RELEASE TAG|
> |Greenland | v1.1.0|
> |Faroe | v1.0.0|
> |Elba | v0.20.0|
> 
> Install SODA Release
> Quick installation from SODA Release:
> 
> Prerequisite
> 
> OS: Ubuntu 16.04 or Ubuntu 18.04
> 
> root user rights is REQUIRED
> .......
> ......
> .......
> 
> Get and Install Release Binaries
> 
> wget https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-<RELEASE_TAG>.tar.gz OR Download the installer binaries from
> https://github.com/sodafoundation/installer/releases/tag/<RELEASE_TAG>

Fixed and updated the PR

File: content/soda-gettingstarted/quickstart.md [hunk 1]
```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
 To have more control of the installaton cofigurations, or try out various configurations refer a detailed installation guide [here](/soda-gettingstarted/installation)
 
 ## Quick Installation and experience of SODA releases
-### Install SODA v1.0.0 Faroe Release Jun 2020
 
-Quick installation from SODA Faroe Release:
+### Install SODA v1.1.0 Greenland Release September 2020
```
Comment (skdwriting) [inline content/soda-gettingstarted/quickstart.md:15]: We can remove this

File: content/soda-gettingstarted/quickstart.md [hunk 2]
```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
...
+Quick installation from SODA Greenland Release:
```
Comment (skdwriting) [inline content/soda-gettingstarted/quickstart.md:16]: We can remove this

File: content/soda-gettingstarted/quickstart.md [hunk 3]
```diff
@@ -11,9 +11,10 @@ This is a quickstart guide for users and new contributors to try and get familia
...
+
```
Comment (skdwriting) [inline content/soda-gettingstarted/quickstart.md:18]: We add a section to mention the releases supported with bulletins. Reverse chronological order
Greenland
Faroe
Elba
Then continue the quick start info\

Please remove the elba from the past release section and update properly

Comment (rajat-soda) [reply content/soda-gettingstarted/quickstart.md:25]: Removed Elba from Past release section.

File: content/soda-gettingstarted/quickstart.md (hunk 2 above)
Comment (rajat-soda) [reply content/soda-gettingstarted/quickstart.md]: Removed

File: content/soda-gettingstarted/quickstart.md (hunk 1 above)
Comment (rajat-soda) [reply content/soda-gettingstarted/quickstart.md]: Removed

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**